src/DotNetCore.CAP.OpenTelemetry/DiagnosticListener.cs

```diff
@@ -57,12 +57,13 @@ public void OnNext(KeyValuePair<string, object?> evt)
                         if (activity != null)
                         {
                             activity.SetTag("message.name", eventData.Operation);
-                            activity.AddEvent(new ActivityEvent("CAP message persistence start...",
-                                DateTimeOffset.FromUnixTimeMilliseconds(eventData.OperationTimestamp!.Value)));
+                            if (eventData.OperationTimestamp is { } timestamp)
```

Comment:
Why OperationTimestamp can be null?

Reply:
Because `OperationTimestamp` is nullable.